docs/source/api/v4/statuses_id.rst

```diff
@@ -0,0 +1,159 @@
+	+-------------+----------+------------------------------------------------------------------------------------------------------+
+	| limit       | no       | Choose the maximum number of results to return                                                       |
+	+-------------+----------+------------------------------------------------------------------------------------------------------+
+	| offset      | no       | The number of results to skip before beginning to return results. Must use in conjunction with limit |
+	+-------------+----------+------------------------------------------------------------------------------------------------------+
+	| page        | no       | Return the n\ :sup:`th` page of results, where "n" is the value of this parameter, pages are         |
+	|             |          | ``limit`` long and the first page is 1. If ``offset`` was defined, this query parameter has no       |
+	|             |          | effect. ``limit`` must be defined to make use of ``page``.                                           |
+	+-------------+----------+------------------------------------------------------------------------------------------------------+
+
+.. code-block:: http
+	:caption: Request Example
+
+	GET /api/4.0/statuses/3 HTTP/1.1
+	Host: trafficops.infra.ciab.test
+	User-Agent: curl/7.47.0
+	Accept: */*
+	Cookie: mojolicious=...
+
+Response Structure
+------------------
+:description: A short description of the status
+:id:          The integral, unique identifier of this status
+:lastUpdated: The date and time at which this status was last modified, in :ref:`non-rfc-datetime`
+:name:        The name of the status
+
+.. code-block:: http
+	:caption: Response Example
+
+	HTTP/1.1 200 OK
+	Access-Control-Allow-Credentials: true
+	Access-Control-Allow-Headers: Origin, X-Requested-With, Content-Type, Accept, Set-Cookie, Cookie
+	Access-Control-Allow-Methods: POST,GET,OPTIONS,PUT,DELETE
+	Access-Control-Allow-Origin: *
+	Content-Type: application/json
+	Set-Cookie: mojolicious=...; Path=/; Expires=Mon, 18 Nov 2019 17:40:54 GMT; Max-Age=3600; HttpOnly
+	Whole-Content-Sha512: dHNip9kpTGGS1w39/fWcFehNktgmXZus8XaufnmDpv0PyG/3fK/KfoCO3ZOj9V74/CCffps7doEygWeL/xRtKA==
+	X-Server-Name: traffic_ops_golang/
+	Date: Mon, 10 Dec 2018 20:56:59 GMT
+	Content-Length: 150
+
+	{ "response": [
+		{
+			"description": "Server is online and reported in the health protocol.",
+			"id": 3,
+			"lastUpdated": "2018-12-10 19:11:17+00",
+			"name": "REPORTED"
+		}
+	]}
+
+``PUT``
+=======
+Creates a Server :term:`Status`.
+
+:Auth. Required: Yes
+:Roles Required: None
+:Permissions Required: STATUS:UPDATE, STATUS:READ
+:Response Type:  Array
+
+Request Structure
+-----------------
+:description:	Return only :term:`Statuses` with this *exact* description                                           |
+:id:			Return only the :term:`Status` with this integral, unique identifier                                 |
```

Comment:
ID is a path parameter that should be in a Path Parameters table, not the body field list.